StandingOvation/StandingOvation.java

```diff
@@ -0,0 +1,107 @@
+import java.io.*;
+import java.util.*;
+import java.io.PrintWriter;
+public class StandingOvation{
+	public static void main(String arg[]){
+		new StandingOvation();

```

Comment:
Instead of perform all the operations when the object is created, use the defined functions after creating the object :)